charts/local-kafka/values.yaml

```diff
@@ -24,6 +24,25 @@ kafka:
   # -- (`str`) Optional value for the kafka cluster pod priority class name
   priorityClassName:
 
+  # -- Optional resource requirements for the Kafka cluster
+  resources:
+    limits:
+      cpu:
+      memory:
+    requests:
+      cpu:
+      memory:
+
+zookeeper:
+  # -- Optional resource requirements for the Zookeeper cluster
+  resources:
+    limits:
+      cpu:
+      memory:
+    requests:
+      cpu:
+      memory:
```

Comment:
set to empty map by default

Reply:
Yes, that's intended. Or are you saying I shouldn't update this file to have the default-null keys at all?